File: app/src/main/java/com/example/cobra/exposit/Notes.java
package com.example.cobra.exposit;

public class Notes {
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Notes() {

    }

    private String date;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    private String userId;

    public Notes(String name, String date) {
        this.name = name;
        this.date = date;
    }

    public Notes(String name, String date, String userId) {
        this.name = name;
        this.date = date;
        this.userId=userId;
    }
}

File: app/src/main/java/com/example/cobra/exposit/Notes.kt
package com.example.cobra.exposit

class Notes {
    var name: String? = null

    var date: String? = null

    var userId: String? = null

    constructor() {

    }

    constructor(name: String, date: String) {
        this.name = name
        this.date = date
    }

    constructor(name: String, date: String, userId: String) {
        this.name = name
        this.date = date
        this.userId = userId
    }
}
